### ISSUE TITLE
Android examples not showing

### ISSUE THREAD

**Issue by AgustinCFZ** [hbldh/bleak#1043]
Hello!,
Please, move this question if this is not the correct place. 
I'm trying to use bleak from android, but the example page is not working:
https://bleak.readthedocs.io/en/examples/kivy

Please, could you fix this?. 

Many thanks!

**Comment by dlech** [comment]
It is in the source tree, not the docs. https://github.com/hbldh/bleak/tree/develop/examples/kivy

Where did you get the link?

**Comment by AgustinCFZ** [comment]
Thanks!. I found the link here:
https://bleak.readthedocs.io/en/latest/backends/android.html

I'm looking a way to use bleak from an android phone to upload a code to the ESP32, but i can't find information on how install bleak on android.
I downloaded an app called "pydroid 3" and tried to execute there a bleak python code, with no success.
If you have a link or any information, i would appreciate it very much!.

Many thanks!
